DEVELOPMENT.md

```diff
@@ -297,6 +298,25 @@ machines.
   _always_ include the reason the note is needed. Remember, more information is better!
 
 
+### Usage Docs
+
+Usage docs should be added to all components and directives. These docs are created by adding a markdown file named to match the component
+or directive:
+
+```
+// component:
+foo.component.ts
+// usage docs:
+foo.component.md
+```
+
+When the documentation published, this usage doc will be automatically consumed by the API docs generation.
+
+When editing or creating usage documentation, add headlines and appropriate content, then run `yarn run docs:toc` to update (or generate)
```

Comment:
@shani-terminus how does the above look?

Reply:
Much better! Explicitly says first you edit, then run the command.